src/components/EasyPill/EasyPill.tsx

```diff
@@ -1,13 +1,36 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import Media from '../Media';
 import EasyDropdown from '../EasyDropdown';
 import Icon from '../Icon';
 import Pill from '../Pill';
+import styles from './EasyPill.module.css';
 
-const EasyPillDropdown = ({ actions, onDelete }) => {
-  if (!actions.length) {
+type EasyPillActions = {
+  children?: React.ReactNode;
+  label: string;
+  onClick: () => void;
+  group?: string;
+};
+
+interface EasyPillProps {
+  actions: EasyPillActions[];
+  checked?: boolean;
+  children: React.ReactNode;
+  onClick?: () => void;
```

Comment:
This component doesn't a top-level `onClick`, right? I think this prop can go away.

Reply:
@pixelbandito I added that `onClick` prop because in the original implementation of the unit test, we are passing onClick to the EasyPill component and I was seeing this TS error `Property 'onClick' does not exist on type 'IntrinsicAttributes & EasyPillProps & { children?: ReactNode; }'`
So should I remove the `onClick` prop from the unit test as well?

Reply:
I think that would be good. No need to test a prop that doesn't isn't used by the component.

While looking at this, I realized that the `...passedProps` for EasyPill` are passed to a `Pill` component.
`Pill` is already using TS, so you should change the `EasyPillProps` interface to extend `PillProps` from that component.

Something like
```
import { PillProps } from '../Pill/Pill';

// ...

interface EasyPillProps extends PillProps {
  // ...
}
```

That will give you the `checked` prop for free, and it will allow consumers to pass any additional props down that are valid for `Pill`.